pydicom/dataset.py

```diff
@@ -589,6 +589,10 @@ def get_item(self, key):
         -------
         pydicom.dataelem.DataElement
         """
+        if isinstance(key, slice):
```

Comment:
Can you add some tests for the addition of slicing to `get_item`?

Reply:
Will do this.